src/test/scala/uk/gov/nationalarchives/notifications/ExportIntegrationSpec.scala

```diff
@@ -0,0 +1,65 @@
+package uk.gov.nationalarchives.notifications
+import java.util.UUID
```

Comment:
Maybe add a line break between package and import statements, for consistency?

Reply:
Fixed